config/settings.yml

```diff
@@ -0,0 +1,91 @@
+rails:
+  time_zone: Eastern Time (US & Canada)
+  active_job:
+    queue_adapter: resque
+  assets:
+    prefix: /assets
+  active_record: {}
+  session_store:
+    type: cookie_store
+    args:
+      key: _markus_session
+      path: <%= ENV.fetch('RAILS_RELATIVE_URL_ROOT') { '/' } %>
+      expire_after: 21 # days
+      secure: false
+      same_site: lax
+  action_mailer:
+    delivery_method: test
+    default_url_options:
+      host: 'localhost:3000'
+    asset_host: 'http://localhost:3000'
+    perform_deliveries: false
+    deliver_later_queue_name: <%= ENV.fetch('MARKUS__QUEUES__DEFAULT') { 'DEFAULT_QUEUE' } %> # Settings.queues.default
```

Comment:
I see what you're doing here. I have a slight preference for making this setting optional, and have `application.rb` set the default queue name if this is `nil`. Feels more consistent with our treatment of the `job` queue settings.